packages/dd-trace/src/llmobs/tagger.js

```diff
@@ -57,6 +58,7 @@ class LLMObsTagger {
     span.setTag(PARENT_ID_KEY, parentId)
   }
 
+  // TODO: similarly for the following `tag` methods, can we pass in a namespaced object instead of the span?
```

Comment:
It should be possible event short-term to have a WeakMap on the LLMObs side that would store tags outside of the span, for example something like `tagger.tagMap`. It could then be improved later on with the Core API but at least the tracing formatter wouldn't need to be modified to know about LLMObs.

Reply:
tried this in 4a19fec0ffc76a39a9104104beb8ff003f57db0b, let me know what you think 😄